site/content/docs/user/configuration.md

```diff
@@ -226,6 +226,10 @@ for persisting data, mounting through code etc.
 {{< codeFromFile file="static/examples/config-with-mounts.yaml" lang="yaml" >}}
 
 
+NOTE: If you are using Docker for Mac or Windows check that the hostPath is
+included in the Preferences -> Resources -> File Sharing.
+[More Info](https://docs.docker.com/docker-for-mac/#file-sharing)
```

Comment:
LGTM, one minor nit is that SIG Docs recommend that we do links like this:

> For more information see the [Docker file sharing guide](....).

Reply:
updated thanks!